tests/phpunit/api/v3/ContactTest.php

```diff
@@ -627,6 +632,9 @@ public function testCreateApiKey($version) {
     $this->assertEquals('A3', $result['values'][$contactId]['first_name']);
     $this->assertEquals('B', $result['values'][$contactId]['last_name']);
     $this->assertEquals('abcd1234', $result['values'][$contactId]['api_key']);
+    // Specifically check get action.
+    $check = $this->callAPISuccessGetSingle('Contact', ['id' => $contactId, 'return' => ['api_key', 'display_name']]);
+    $this->assertEquals('abcd1234', $check['api_key']);
```

Comment:
It fails here & it turns out the BAO_Query doesn't return it to the api for the remove routine to work